app/components/Map.js

```diff
@@ -15,7 +15,7 @@ import baseImg from '../assets/redPin.png';
 import targetImg from '../assets/blackPin.png';
 import { PinCallout } from './PinCallout';
 import PinEditButton from './PinEditButton';
-import { myCurrLoc, currLoc } from '../lib/db/db';
+import * as geoAction from '../lib/orientation/utils';
```

Comment:
@tiffeli We lost the `myCurrLoc` and `currLoc` variables here, but they're still in use on line 64 (throws an error).